test/tools/d_do_test.d

```diff
@@ -638,7 +638,7 @@ bool compareOutput(string output, string refoutput, const ref EnvData envData)
                 break;
             }
             // Match against OS or model
-            else if (conditional[0].among(envData.os, envData.model))
+            else if (conditional[0].among(envData.os, envData.model, envData.model[0 .. min(2, $)]))
```

Comment:
Might want to update the comment to say `Match against OS or model. Match 32mscoff as 32`